README.md

```diff
@@ -114,6 +113,13 @@ Js.log "Hello BuckleScript!"
 
 #### Use string interpolation
 
+```ml
+let () =
+  let world = {j|World|j} in
```

Comment:
This would be better as a normal string, otherwise it falsely implies that string interpolation parameters need to be "special" strings.